.github/workflows/build.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
                      "gcc_release") GN_ARGS='is_debug=false';;
                      "clang") GN_ARGS='is_clang=true';;
                      "mbedtls") GN_ARGS='chip_crypto="mbedtls"';;
-                     "clang_experimental") GN_ARGS='is_clang=true chip_enable_interaction_model=true';;
```

Comment:
Is this change intentional?

Reply:
Yes -- Temporarily disable IM in experimental build since these API are used in ember IM, and we need to update ember in correspondence in separate PR and avoid large PR, the ticket is in https://github.com/project-chip/connectedhomeip/issues/6021